shell/common/shell.cc

```diff
@@ -2023,6 +2037,71 @@ bool Shell::OnServiceProtocolReloadAssetFonts(
   return true;
 }
 
+void Shell::AddView(int64_t view_id) {
+  TRACE_EVENT0("flutter", "Shell::AddView");
+  FML_DCHECK(is_set_up_);
+  FML_DCHECK(task_runners_.GetPlatformTaskRunner()->RunsTasksOnCurrentThread());
+  if (view_id == kFlutterImplicitViewId) {
+    FML_DLOG(WARNING)
```

Comment:
This seems like a logical bug that shouldn't happen? If so, prefer `ERROR` so that it shows up in logs.

```suggestion
    FML_DLOG(ERROR)
```